app/src/main/java/io/btkelly/gnagtest/MainActivity.java

```diff
@@ -1,13 +1,20 @@
 package io.btkelly.gnagtest;
 
-import android.support.v7.app.AppCompatActivity;
+import android.app.Activity;
 import android.os.Bundle;
+import android.util.Log;
 
-public class MainActivity extends AppCompatActivity {
+public class MainActivity extends Activity {
 
     @Override
     protected void onCreate(Bundle savedInstanceState) {
         super.onCreate(savedInstanceState);
         setContentView(R.layout.activity_main);
+
+        try{
```

Comment:
<b>Reporter: </b>Checkstyle<br /><b>Violation: </b>WhitespaceAroundCheck<br /><b>Notes: </b>'{' is not preceded with whitespace.